Makefile

```diff
@@ -9,7 +9,10 @@ black:
 flake8:
 	@poetry run flake8 --ignore=E501,W501,E231,W503
 
-check: black flake8
+mypy:
```

Comment:
@Stannislav would you mind adding `mypy` to [.PHONY](https://github.com/scanapi/scanapi/blob/main/Makefile#L33)?

Reply:
Ah yes, of course, totally missed it!

Done in 8363fed.